apps/homescreen/js/app.js

```diff
@@ -436,7 +443,7 @@ const SETTINGS_VERSION = 0;
       this.handleEvent({ type: 'scroll' });
     },
 
-    snapScrollPosition: function(bias) {
+    refreshMetrics: function(snap, bias) {
```

Comment:
Yeah, good call - I think I didn't because the snap part relies on information from the calculation part, but I could easily just return that information. Will do.